test/models/tutorial_enrolment_model_test.rb

```diff
@@ -0,0 +1,25 @@
+require "test_helper"
+
+class TutorialEnrolmentModelTest < ActiveSupport::TestCase
+  def test_default_create
+    tutorial_enrolment = FactoryGirl.create(:tutorial_enrolment)
+    assert tutorial_enrolment.valid?
+  end
+
+  def test_specific_create
+    project = FactoryGirl.create(:project)
+    tutorial = FactoryGirl.create(:tutorial)
+    tutorial_enrolment = FactoryGirl.create(:tutorial_enrolment, project: project, tutorial: tutorial)
```

Comment:
Shouldn't we be testing how "we" enrol projects in tutorials?

Then also test all of the above things that I mentioned as missing validations.